src/components/Notifications/NotificationTypes/BidReceivedNotification.tsx

```diff
@@ -60,13 +61,23 @@ const BidReceivedNotification = ({
       }}
       timestamp={notification.timestamp}
       isNew={!notification.read}
+      locale={locale}
     >
       <p className="dcl notification-item__content-title">
         {i18N[locale].title}
       </p>
       <p className="dcl notification-item__content-description">
         {i18N[locale].description(
-          formatMana(notification.metadata.price),
+          <Mana
+            inline
+            network={
+              notification.metadata.network === 'polygon'
+                ? Network.MATIC
+                : Network.ETHEREUM
```

Comment:
What do you think about sending this logic inside the component?

Reply:
hmm what do u mean? `<Mana />` component expects `Network` enum from `@dcl/schemas` and this enum is not equivalent to what it comes from the backend which is `"polygon" | "ethereum"`.